src/main/flight/mixer_init.c

```diff
@@ -52,6 +54,11 @@ PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .crashflip_motor_percent = 0,
     .crashflip_expo = 35,
     .mixer_type = MIXER_LEGACY,
+    .rpm_limiter_p = 25,
```

Comment:
#ifdef ?

Reply:
`./src/main/flight/mixer_init.c:57:1: error: embedding a directive within macro arguments is not portable [-Werror]
   57 | #ifdef USE_RPM_LIMITER
`
`./src/main/flight/mixer_init.c:63:1: error: embedding a directive within macro arguments is not portable [-Werror]
   63 | #endif
`
Ideas on how to fix this @limonspb ?